frontend/src/components/.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
no need to ignore it in a subfolder once it's ignored in the root .gitignore file